views/js/controller/creator/creator.js

```diff
@@ -98,6 +116,7 @@ define([
             //back button
             $('#authoringBack').on('click', function(e){
                 e.preventDefault();
+                $(document).trigger('creatorclose');
```

Comment:
I know we've been attaching events to the document for the item creator. But since it is not the best practice anymore, could we not find a better way to manage events. I way thinking about an eventified object that is shared everywhere we should give us more control other the events.